components/editor/TimesheetEntryEditor.tsx

```diff
@@ -96,16 +97,43 @@ const DayEditor = ({
                                 date={dateStr}
                                 timeCategories={timeCategories}
                             />
+                            <Link
+                                key={`updateEntryEditor-${entry.id}`}
+                                onClick={() => setInEditState(!inEditState)}
+                                style={{
+                                    marginRight: ".5rem",
+                                    fontWeight: "bold",
+                                }}
+                            >
+                                Close
+                            </Link>
                             <Link
                                 key={`delete-${entry.id}`}
                                 onClick={() => deleteTimesheetEntry(id)}
+                                style={{
+                                    marginLeft: ".5rem",
+                                    fontWeight: "bold",
+                                }}
                             >
-                                delete
+                                Delete
                             </Link>
                         </div>
                     )
                 }
-                return null
+                return (
+                    <div key={`updateEntryEditor-${entry.id}`}>
+                        <Link
+                            key={`updateEntryEditor-${entry.id}`}
+                            onClick={() => setInEditState(!inEditState)}
+                            style={{
+                                marginRight: ".5rem",
+                                fontWeight: "bold",
+                            }}
+                        >
+                            Inspect
```

Comment:
I don't think `inspect` is very useful content for this `Link`. I think something like
```
7.5 hours ${project.name}
```
would be a more useful thing to display.